Packs/Reco/Integrations/Reco/Reco.py

```diff
@@ -533,6 +620,9 @@ def main() -> None:
             email_address = demisto.args()["email_address"]
             result = add_risky_user_label(reco_client, email_address)
             return_results(result)
+        elif command == "reco-get-assets-user-has-access-to":
+            result = get_assets_by_owner(reco_client, demisto.args()["asset_owner"], demisto.args()["only_sensitive"])
```

Comment:
I can't find the definition of the `get_assets_by_owner` method. Did you mean `get_assets_user_has_access`?

Reply:
Fixed everything here
https://github.com/demisto/content/pull/25391

Reply:
Sounds good